homeassistant/components/hive/manifest.json

```diff
@@ -4,7 +4,7 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/hive",
   "requirements": [
-    "pyhiveapi==0.4.2"
+    "pyhiveapi==0.5.1"
```

Comment:
Please add a link to a changelog or GitHub commit compare view for the version bump in the PR description. Thanks!

Reply:
Link added to PR description